img/plus.svg

```diff
@@ -0,0 +1 @@
+<svg fill="#26e07f" xmlns="http://www.w3.org/2000/svg"  viewBox="0 0 24 24" width="24px" height="24px"><path fill-rule="evenodd" d="M 11 2 L 11 11 L 2 11 L 2 13 L 11 13 L 11 22 L 13 22 L 13 13 L 22 13 L 22 11 L 13 11 L 13 2 Z"/></svg>
```

Comment:
This should be available already as a global icon class `icon-add` https://docs.nextcloud.com/server/latest/developer_manual/html_css_design/icons.html

Otherwise we can make use of https://github.com/robcresswell/vue-material-design-icons for pulling in icons.